src/daemon/globals.c

```diff
@@ -0,0 +1,13 @@
+#include "globals.h"
+#include "plugin.h"
+/*
+ * Global variables
+ */
+char hostname_g[DATA_MAX_NAME_LEN];
+const int hostname_g_size = sizeof (hostname_g);
+cdtime_t interval_g;
+int  pidfile_from_cli = 0;
```

Comment:
`pidfile_from_cli` is unused, likely a remnant of an unrelated change. Please remove.